task_processing/plugins/mesos/execution_framework.py

```diff
@@ -140,11 +135,13 @@ def _background_check(self):
                                 task_event(
                                     task_id=task_id,
                                     terminal=True,
-                                    timestamp=time.time(),
+                                    timestamp=time_now,
```

Comment:
It might not make any difference because the granularity is second. But I think time.time() is more accurate than time_now?

Reply:
the whole loop should be done in milliseconds while involved timeout are expressed in hundreds/thousands of seconds, cached `time.time()` call is fine here. besides, `time_now` is what the logic of conditionals is based on - it makes sense to report it with any resulting events.